cmd/lookoutd/web.go

```diff
@@ -50,13 +50,13 @@ func (c *WebCommand) Execute(args []string) error {
 
 	ghConfg := conf.Providers.Github
 	if ghConfg.ClientID == "" {
-		return fmt.Errorf("provider github client_id is required")
+		return fmt.Errorf("Missing field in configuration file: provider github client_id is required")
```

Comment:
I seem to remember that it's a good practice to start the error message with a lowercase.

Reply:
absolutely true. Though in this particular case it's not that important because it's top-level error.

Reply:
Yes. I made it uppercase for consistency with the other top-level error messages above this one. Maybe we could change all of the error messages to lowercase at some point.